Lib/test/test_ctypes/test_wintypes.py

```diff
@@ -38,6 +41,22 @@ def test_variant_bool(self):
         vb.value = []
         self.assertIs(vb.value, False)
 
+    def test_signess(self):
```

Comment:
I had to check the spelling myself, but the right word appear to be ["signedness"](https://en.wikipedia.org/wiki/Signedness).

Reply:
Fixed.